src/govuk/settings/_warnings.scss

```diff
@@ -0,0 +1,50 @@
+////
+/// @group settings/warnings
+////
+
+/// Suppressed warnings map
+///
+/// This map is used to determine which deprecation warnings to **not** show
+/// to users when compiling sass. This is in place for codebases that don't
+/// have the necessary capacity to upgrade and remove the deprecation,
+/// particularly if said deprecation is significant eg: removal of compatibility
+/// with legacy libraries such as govuk_elements.
+///
+/// You can add to this map and define which warnings to suppress by appending to
+/// it using the warning key, found in the warning message. For example:
+///
+/// @example scss:
+///   // warning message:
+///   //  $foobar is no longer supported. To silence this warning, update
+///   //  $govuk-suppressed-warnings with key: "foobar"
+///   $govuk-suppressed-warnings: append($govuk-suppressed-warnings, "foobar");
+///
+/// @type Map
```

Comment:
There is no key-value association by the description of how it's used, so it's more a [List](https://sass-lang.com/documentation/values/lists) than a[Map](https://sass-lang.com/documentation/values/maps)